math-libs/BLAS/CMakeLists.txt

```diff
@@ -36,6 +36,51 @@ therock_cmake_subproject_provide_package(hipBLAS-common hipblas-common lib/cmake
 therock_cmake_subproject_activate(hipBLAS-common)
 list(APPEND _blas_subproject_names hipBLAS-common)
 
+
+##############################################################################
+# rocRoller
+##############################################################################
+
+therock_cmake_subproject_declare(rocRoller
+  EXTERNAL_SOURCE_DIR "rocRoller"
+  BACKGROUND_BUILD
+  CMAKE_LISTS_RELPATH "next-cmake"
+  CMAKE_ARGS
+    -DHIP_PLATFORM=amd
+    -DROCROLLER_ENABLE_YAML_CPP=ON
+    -DROCROLLER_ENABLE_CLIENT=OFF
+    -DROCROLLER_BUILD_TESTING=${THEROCK_BUILD_TESTING}
+    -DROCROLLER_ENABLE_CATCH=OFF
+    -DROCROLLER_BUILD_SHARED_LIBS=OFF
+  CMAKE_INCLUDES
+    therock_explicit_finders.cmake
+  COMPILER_TOOLCHAIN
+    amd-hip
+  BUILD_DEPS
+    rocm-cmake
+    therock-boost
+    therock-fmt
+    therock-googletest
+    therock-host-blas
```

Comment:
You might need to move 
```suggestion
```
This needs to be moved to runtime deps, as OpenBLAS builds shared libraries which you rather need during runtime and not only when building.